app-office/libreoffice/libreoffice-5.1.4.2.ebuild

```diff
@@ -265,7 +265,7 @@ pkg_pretend() {
 	if [[ ${MERGE_TYPE} != binary ]]; then
 		check-reqs_pkg_pretend
 
-		if [[ $(tc-getCC) == clang ]] ; then
+		if [[ $(tc-getCC) == *clang* ]] ; then
```

Comment:
I am unfortunately not the *ebuild* maintainer for this project, and I do not know who is.  Should I close my pull request?